kubernetes/resource_kubernetes_namespace.go

```diff
@@ -56,6 +62,31 @@ func resourceKubernetesNamespaceCreate(ctx context.Context, d *schema.ResourceDa
 	log.Printf("[INFO] Submitted new namespace: %#v", out)
 	d.SetId(out.Name)
 
+	if d.Get("wait_for_default_service_account").(bool) {
+		log.Printf("[DEBUG] Waiting for default service account to be created")
+		// FIXME: Since the creation of the Default ServiceAccount is asynchronous to the Namespace
+		// We need to wait some period of time, using something like d.Timeout(schema.TimeoutCreate)
+		// is not long enough for a Kind cluster to pass without timing out (about 10 seconds).
+		// Using an undefined word here forces the timeout to use the defaultTimeout of 20mins.
+		// I do not consider this production ready.
```

Comment:
I'm not sure I understand the logic here. The schema for the `kubernetes_namespace` resource doesn't specify a create timeout, so if you use schema.TimeoutCreate you'll get the default of 20 minutes anyway.